2021-12-15 Shiny shiny little star/app.R

```diff
@@ -0,0 +1,122 @@
+library(DT)
+library(highcharter)
+library(readxl)
+library(shiny)
+library(tidyr)
+library(dplyr)
+library(stringr)
+library(tidytext)
+library(googlesheets4)
+library(textdata)
+library(tidyr)
+library(lubridate)
+library(RColorBrewer)
+library(wordcloud2)
+library(tm)
+library(tidyverse)
+library(qdapRegex)
+library(reshape2)
+library(chron)
+
+#~~~~WARNING: before running this app, set the correct current month on the data page~~~~
+
+#~~~~header function~~~~
+mbie_header <- function()
+  div(
+    fluidRow(
+      column(
+        8,
+        div(
+          br(),
+          h2("Christmas planninnnnng!!"),
+          br()
+        )),
+      column(
+        4,
+        br(),
+        img(src = "bsa_logo.svg", width = 300),
+        br()
+      )
+    ),
+    div(class = "mbie-topbar"))
+
+
+source('./Pages/ui/ui.R', local = TRUE)

```

Comment:
I think we can set this up as an R project so everything is relative?

Does that get around using `local = TRUE` in all the `source` calls?